pkg/fqdn/matchpattern/matchpattern.go

```diff
@@ -70,6 +70,11 @@ func ToAnchoredRegexp(pattern string) string {
 	pattern = strings.TrimSpace(pattern)
 	pattern = strings.ToLower(pattern)
 
+	// handle the ** match-all case.
+	if pattern == "**" {
```

Comment:
If we end up returning the same as for the `pattern == "*"` case, I'd make it 

`if pattern == "*" || pattern == "**" {` to avoid the duplication.

Reply:
Good point, thank you!